src/components/auth/profile/CertificatesTab.tsx

```diff
@@ -1,11 +1,157 @@
+import VisibilityIcon from '@mui/icons-material/Visibility'
+import ArrowForwardIcon from '@mui/icons-material/ArrowForward'
+const useStyles = makeStyles({
+  h3: {
+    fontFamily: 'Montserrat',
+    fontStyle: 'normal',
+    fontWeight: '500',
+    fontSize: '25px',
+    lineHeight: '116.7%',
+    margin: '0',
+  },
+  thinFont: {
+    fontFamily: 'Montserrat',
+    fontStyle: 'normal',
+    fontWeight: 400,
+    fontSize: '24px',
+    lineHeight: '123.5%',
+    letterSpacing: '0.25px',
+    color: '#000000',
+    margin: 0,
+  },
+  smallText: {
+    fontFamily: 'Lato, sans-serif',
+    fontStyle: 'normal',
+    fontWeight: '500',
+    fontSize: '20px',
+    lineHeight: '160%',
+    letterSpacing: '0.15px',
+  },
+})
 
 function CertificatesTab(props: { value: number; index: number }) {
   const { value, index } = props
+  const classes = useStyles()
+  const { data = { donations: [], total: 0 } } = useUserDonations()
+  const [fromDate, setFromDate] = React.useState(new Date())
+  const [toDate, setToDate] = React.useState(new Date())
 
   return (
     <Tab value={value} index={index}>
-      <h2>История на сертификати</h2>
+      <Box
+        sx={{
+          backgroundColor: 'white',
+          padding: '10px 30px',
+          margin: '10px 0 0 0',
+        }}>
+        <h3 className={classes.h3}>История на сертификати</h3>
+      </Box>
+      <Box
+        sx={{
+          backgroundColor: 'white',
+          flexGrow: 1,
+          padding: '10px 30px',
+        }}>
+        <Box>
+          <h3 className={classes.thinFont}>Онлайн дарения</h3>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'baseline',
+            justifyContent: 'space-between',
+          }}>
+          <span className={classes.smallText}>Покажи:</span>
+          <Box>
+            <Checkbox defaultChecked />
+            <span className={classes.smallText}>еднократни</span>
+          </Box>
+          <Box>
+            <Checkbox defaultChecked />
+            <span className={classes.smallText}>месечни</span>
+          </Box>
+          <LocalizationProvider dateAdapter={AdapterDateFns}>
+            <span className={classes.smallText}>от дата</span>
+            <DesktopDatePicker
+              label="от дата"
+              inputFormat="MM/dd/yyyy"
+              value={fromDate}
+              onChange={(date) => setFromDate(date as Date)}
+              renderInput={(params) => <TextField {...params} />}
+            />
+            <span className={classes.smallText}>до дата</span>
+            <DesktopDatePicker
+              label="до дата"
+              inputFormat="MM/dd/yyyy"
+              value={toDate}
+              onChange={(date) => setToDate(date as Date)}
+              renderInput={(params) => <TextField {...params} />}
+            />
+          </LocalizationProvider>
+        </Box>
+        {data.donations.length ? (
+          <TableContainer>
```

Comment:
This table component can be extracted as a separate component, it will reduce the visual nesting when using a condition.

Reply:
This can be done in a next iteration so we don't hold this in the pipeline